### PR TITLE
[MNG-7463] Improve documentation about deprecation in Mojo

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: content/xdoc/developers/mojo-api-specification.xml [hunk 1]
```diff
@@ -588,9 +588,9 @@ under the License.
           <td><code>deprecated</code></td>
           <td><code>@deprecated &lt;deprecated-text&gt;</code></td>
           <td>No</td>
-          <td>Specify the version when the Mojo was deprecated to the API. Similar to Javadoc deprecated.
-            This will trigger a warning when a user tries to configure a parameter
-            marked as deprecated.</td>
+          <td>Description with the reason of Mojo depreciation. Similar to Javadoc deprecated.
```
Comment (michael-o) [inline content/xdoc/developers/mojo-api-specification.xml:591]: deprecation

File: content/xdoc/developers/mojo-api-specification.xml [hunk 2]
```diff
@@ -588,9 +588,9 @@ under the License.
...
+            This will trigger a warning when a user tries to use Mojo marked as deprecated.
```
Comment (michael-o) [inline content/xdoc/developers/mojo-api-specification.xml:592]: use a Mojo

File: content/xdoc/developers/mojo-api-specification.xml [hunk 3]
```diff
@@ -731,9 +731,9 @@ under the License.
           <td><code>deprecated</code></td>
           <td><code>@deprecated &lt;deprecated-text&gt;</code></td>
           <td>No</td>
-          <td>Specify the version when the Mojo was deprecated to the API. Similar to Javadoc deprecated.
-            This will trigger a warning when a user tries to configure a parameter
-            marked as deprecated.</td>
+          <td>Description with the reason of parameter depreciation. Similar to Javadoc deprecated.
```
Comment (michael-o) [inline content/xdoc/developers/mojo-api-specification.xml:734]: same here

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by slawekjaranowski** [comment]
@michael-o Fixed according to comments.

File: content/xdoc/developers/mojo-api-specification.xml [hunk 4]
```diff
@@ -588,9 +588,9 @@ under the License.
           <td><code>deprecated</code></td>
           <td><code>@deprecated &lt;deprecated-text&gt;</code></td>
           <td>No</td>
-          <td>Specify the version when the Mojo was deprecated to the API. Similar to Javadoc deprecated.
-            This will trigger a warning when a user tries to configure a parameter
-            marked as deprecated.</td>
+          <td>Description with the reason of Mojo depreciation. Similar to Javadoc deprecation.
```
Comment (michael-o) [inline content/xdoc/developers/mojo-api-specification.xml:591]: Still typo

File: content/xdoc/developers/mojo-api-specification.xml [hunk 5]
```diff
@@ -731,9 +731,9 @@ under the License.
           <td><code>deprecated</code></td>
           <td><code>@deprecated &lt;deprecated-text&gt;</code></td>
           <td>No</td>
-          <td>Specify the version when the Mojo was deprecated to the API. Similar to Javadoc deprecated.
-            This will trigger a warning when a user tries to configure a parameter
-            marked as deprecated.</td>
+          <td>Description with the reason of parameter depreciation. Similar to Javadoc deprecation.
```
Comment (michael-o) [inline content/xdoc/developers/mojo-api-specification.xml:734]: Same here

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by michael-o** [comment]
See: https://www.oxfordlearnersdictionaries.com/definition/english/deprecation?q=deprecation

**Comment by slawekjaranowski** [comment]
I see ... such a similar words  ... 😄

**Review by michael-o — APPROVED** [review]

**Pull request merged**